custom_code/views.py

```diff
@@ -40,7 +40,16 @@
 
 import plotly.graph_objs as go
 from tom_dataproducts.models import ReducedDatum, DataProduct
-from custom_code.templatetags.custom_code_tags import airmass_collapse, lightcurve_collapse, spectra_collapse, lightcurve_fits, lightcurve_with_extras, get_best_name, dash_spectra_page, scheduling_list_with_form, smart_name_list
+from custom_code.templatetags.custom_code_tags import (
```

Comment:
Stylistically I don't like importing this many functions. I would probably do a from custom_code.templatetags import custom_code_tags and add a custom_code_tags. in front of the function calls.